### PR TITLE
Perl6Doc: Switch fetch URL to https

### PR DESCRIPTION
**Pull request by moollaza**
## Description of new Instant Answer, or changes
- Ensure the fetch URL for perl6.org uses `https`.

## People to notify
@sahildua2305  @GuiltyDolphin 

---

Instant Answer Page: https://duck.co/ia/view/perl6doc

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by moollaza** [comment]
@sahildua2305 please test + review + merge

File: lib/fathead/perl6doc/fetch.sh [hunk 1]
```diff
@@ -4,5 +4,5 @@
 # production usage).
 mkdir -p download
 cd download
-wget -np -nc -r -l 2 http://doc.perl6.org/
+wget -np -nc -r -l 2 https://doc.perl6.org/
```
Comment (GuiltyDolphin) [inline lib/fathead/perl6doc/fetch.sh]: Looks like this redirects to `https://docs.perl6.org`

**Comment by moollaza** [comment]
@GuiltyDolphin good catch. Updated again 👍

**Comment by sahildua2305** [comment]
Sorry for the delay on this. I will test this soon!

**Comment by moollaza** [comment]
@sahildua2305 ping!

**Pull request merged**